boards/nucleo144-f429/include/periph_conf.h

```diff
@@ -26,25 +26,34 @@ extern "C" {
 #endif
 
 /**
- * @name    Clock system configuration
+ * @name Clock settings
+ *
+ * @note This is auto-generated from
+ *       `cpu/stm32_common/dist/clk_conf/clk_conf.c`
  * @{
  */
+/* give the target core clock (HCLK) frequency [in Hz],
+ * maximum: 180MHz */
+#define CLOCK_CORECLOCK      (120000000U)
```

Comment:
with coreclock at 180MHz, it's not possible to generate the 48MHz from the PLL. Updated to 168MHz, though.